sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkBatchClient.cs

```diff
@@ -28,5 +31,145 @@ public SparkBatchClient(Uri endpoint, string sparkPoolName, TokenCredential cred
                   options.VersionString)
         {
         }
+
+        /// <summary> Create new spark batch job. </summary>
+        /// <param name="sparkBatchJobOptions"> Livy compatible batch job request payload. </param>
+        /// <param name="detailed"> Optional query param specifying whether detailed response is returned beyond plain livy. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        internal virtual async Task<Response<SparkBatchJob>> CreateSparkBatchJobAsync(SparkBatchJobOptions sparkBatchJobOptions, bool? detailed = null, CancellationToken cancellationToken = default)
```

Comment:
Why is this method needed? Doesn't StartCreateSparkBatchJobAsync do everything that is needed?

Reply:
Called later by StartCreateSparkBatchJobAsync. 

Each of the StartCreateFoo methods call CreateFoo to do the major work. I had to copy them into the customization layer just to make them internal.

Reply:
So why do we have a separate diagnostic scope in both places? Doesn't StartCreate.. call right into this?